examples/confluence-data-retrieval-example/20_fetch_confluence_space.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021-2024 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import logging
+import os
+
+from langchain_community.document_loaders import ConfluenceLoader
+from vdk.api.job_input import IJobInput
+
+log = logging.getLogger(__name__)
+
+
+def fetch_confluence_space(url, token, space_key):
+    try:
+        # For more info regarding the LangChain ConfluenceLoader:
+        # https://python.langchain.com/docs/integrations/document_loaders/confluence
+        loader = ConfluenceLoader(url=url, token=token)
+        documents = loader.load(
+            space_key=space_key, include_attachments=True, limit=20, max_pages=20
```

Comment:
here why do we have this limit?

Reply:
It's an example, I didn't want to make it too big but also wanted to show that we could limit it to a number we want.